Makefile

```diff
@@ -46,6 +46,7 @@ all: build
 
 
 build: $(FLASHABLEZIP)
+
```

Comment:
I'm not sure if this line can be there. Are you sure It doesn't conflict with Make's strict syntax?

Reply:
Oh, right, it just looked dangerous in a quick view, but it's an "alias" target (what you call rule). You are right, it looks more clear that way. I'll add another in L116 for the ```release``` target.